lms/djangoapps/courseware/tests/test_credit_requirements.py

```diff
@@ -133,6 +133,26 @@ def test_credit_requirements_not_eligible(self):
         )
         self.assertContains(response, "Verification Failed")
 
+    def test_credit_requirements_honor_enrollment(self):
+        # Test the progress table is not displayed to the honor students.
+        credit_api.set_credit_requirement_status(
+            self.user.username, self.course.id,
+            "reverification", "midterm",
+            status="failed", reason={}
+        )
+        self.enrollment.mode = 'honor'
```

Comment:
@aamir-khan , here's a link to the ddt docs: http://ddt.readthedocs.org/en/stable/example.html

Here's one of many places we use it: https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/management/commands/tests/test_create_course.py#L47-L70. But you can grep "ddt" and find other places.